scripts/issue265-shim-selftest.sh

```
@@ -0,0 +1,291 @@
+
+[[ -x "$SHIM" ]] || {
+    echo "FATAL: shim not found or not executable: $SHIM" >&2
+    exit 1
+}
+command -v timeout >/dev/null 2>&1 || {
+    echo "FATAL: timeout is required for the Linux shim self-test" >&2
+    exit 1
+}
+
+# The exact production vectors (must match the shim's readonly constants).
+SEARCH_QUERY_BODY='query($searchQuery: String!, $first: Int!) { search(type: ISSUE, query: $searchQuery, first: $first) { nodes { ... on Issue { id number title state author { login } updatedAt assignees(first: 10) { nodes { login } } labels(first: 20) { nodes { name } } issueType { name } milestone { title } comments { totalCount } } } pageInfo { hasNextPage endCursor } } }'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)